backend/src/parser.py

```diff
@@ -0,0 +1,9 @@
+import sys
```

Comment:
Please add a file level comment to the top of the new file similar to the other files